cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -140,6 +140,14 @@
           "description": "The user's login name. Required otherwise user creation will be skipped for this user.",
           "type": "string"
         },
+        "doas": {
+          "description": "List of doas rules to add for a user.",
```

Comment:
Since this key is only supported on distros that have doas in place of sudo, could we document something to say that? That will help avoid user confusion.

Maybe something like:
```
List of doas rules to add for a user. Doas must be installed for rules to take effect.
```

Reply:
Done